.circleci/config.yml

```diff
@@ -1,4 +1,4 @@
-version: 2
+version: 2.1
```

Comment:
What do we get with that?

Reply:
[The version 2.1 introduces commands](https://discuss.circleci.com/t/circleci-2-1-config-overview/26057), that I am using to separate the app setup steps from the steps to run e2e tests.